perf/istio-install/setup_istio.sh

```diff
@@ -108,18 +108,94 @@ function install_extras() {
   # Redeploy, this time with the Prometheus resource created
   helm template --set domain="${domain}" "${WD}/base" | kubectl apply -f -
   # Also deploy relevant ServiceMonitors
-  "${release}/bin/istioctl" manifest generate --set profile=empty --set addonComponents.prometheusOperator.enabled=true -d "${release}/manifests" | kubectl apply -f -
+  if [[ -f "${release}/samples/addons/extras/prometheus-operator.yaml" ]];then
+     kubectl apply -f "${release}/samples/addons/extras/prometheus-operator.yaml" -n istio-system
```

Comment:
wait, this is shared by benchmark perf test as well, if you install prometheus using samples/addons, what should we setup in benchmark test? i mean this PR: https://github.com/istio/tools/pull/1111

Reply:
this is not prometheus itself, this is the prometheusOperator related service monitors. Prometheus itself is installed at line 109